deploy/nginx.conf

```diff
@@ -1,11 +1,18 @@
 server {
-    listen 80;
-    server_name localhost 0.0.0.0;
+   listen 80;
+    server_name explorer.continuum.io;
     client_max_body_size 100M;
 
-    location / {
-        proxy_pass http://0.0.0.0:8000/;
-    }
+    auth_basic "Restricted";
+    auth_basic_user_file /etc/nginx/.htpasswd;
+
+   location / {
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header Host $host;
+      proxy_pass  http://unix:/home/vagrant/gunicorn.sock;
+      proxy_set_header Authorization "Basic a2luZzppc25ha2Vk";
```

Comment:
We need to have it on explorer.continuum.io. In theory we could have a version of nginx.conf that doesn't have it and a version that does and decide on deploy-time which one it should be. Is there value in removing the HTTP authentication for development?